frontend/ui/widget/scrollhtmlwidget.lua

```diff
@@ -89,7 +92,10 @@ end
 
 function ScrollHtmlWidget:scrollToRatio(ratio)
     ratio = math.max(0, math.min(1, ratio)) -- ensure ratio is between 0 and 1 (100%)
-    local page_num = 1 + Math.round((self.htmlbox_widget.page_count - 1) * ratio)
+    local page_num = 1 + math.floor((self.htmlbox_widget.page_count) * ratio)
```

Comment:
What does this change do precisely?

Reply:
Precisely? It makes it works :) 
I added these methods scrollToRatio initially as some generic stuff, but it was only needed until now to do scrollToRatio(1) (100%) in DictQuickLookup to go at the bottom of previous result when paging back.
I never really tested it with other real ratios between 0 and 1, and never wondered what has to be positionned at this ratio: the top? the middle?
Math.round() I think could possibly position at the next page, so the position in the scrollbar you tapped on might be not in the black part you'll be then displaying.
With math.floor(), we go to the nearest top of page before your tap position, so your tap position is in the black bar.
(I'm getting old, more and more of these arithmetic stuff, I get to the end of them with trial and errors... my first intuition rarely works out... The +1 / -1 was that. Tested with a dict result spanning 3 pages, then 4, seemed to work, had a drink :)